### DISCUSSION TITLE
相册 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#302] [Announcements]
# 相册 | Zi-Zi's Journey

Zi-Zi's Journey

https://gz-metal-cell.github.io/galleries/

**Comment by promefire** [comment]
鼠标移过去图片会动是怎么实现的

**Reply by GZ-Metal-Cell** [reply]
css下img:hover{}属性，然后transform，transition啥的，具体的f12看一下

**Reply by promefire** [reply]
太强了！！偷了偷了

**Reply by GZ-Metal-Cell** [reply]
嚯嚯 pro神的相册建设得咋样了 我想看😭

**Reply by promefire** [reply]
还没开始
😭

**Reply by GZ-Metal-Cell** [reply]
胡说😭pro 神已经建好了文件夹😭太强了😭pro 神😭
